drivers/vmwarevsphere/vsphere.go

```diff
@@ -162,6 +163,7 @@ func (d *Driver) GetMachineId() (string, error) {
 	return d.MachineId, nil
 }
 
+
```

Comment:
nit

Reply:
I will bump this to more than a nit. It should be removed.

Reply:
done